models/reddit_ads__url_report.sql

```diff
@@ -57,6 +57,8 @@ joined as (
         sum(report.impressions) as impressions,
         sum(report.spend) as spend
 
+        
+
```

Comment:
Can we add the `{{ fivetran_utils.persist_pass_through_columns(pass_through_variable='reddit_ads__ad_passthrough_metrics', transform = 'sum') }}` macro to this model.

Reply:
oops, forgot to add here-- added